# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/gcc/hotfix.patch.uclibc
# Copyright (C) 2023 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- gcc-13.1.0/libstdc++-v3/src/c++20/tzdb.cc.vanilla	2023-05-25 00:45:21.569447104 +0200
+++ gcc-13.1.0/libstdc++-v3/src/c++20/tzdb.cc	2023-05-25 00:45:54.347450473 +0200
@@ -633,10 +633,10 @@
 	void decrement() { }
 
 	// Use a mutex to synchronize all access to the infos vector.
-	mutex infos_mutex;
+	//mutex infos_mutex;
 
-	void lock() { infos_mutex.lock(); }
-	void unlock() { infos_mutex.unlock(); }
+	void lock() { /*infos_mutex.lock();*/ }
+	void unlock() { /*infos_mutex.unlock();*/ }
       };
 
 #if defined __GTHREADS && __cpp_lib_atomic_wait
